### PR TITLE
[MAINTENANCE] DOCS integration tests have moved to a new pipeline 

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
- Integration tests for documentation currently live in the main `azure-pipelines.yml` as a separate stage. 
- This PR allows for the docs tests to live in a separate pipeline, where we would have more control over their cadence

### CODE REVIEW

File: azure-pipelines-docs-integration.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+trigger:
+  branches:
+    include:
+    - pre_pr_docs-*
+    #- develop
+    #- main
```
Comment (Shinnnyshinshin) [inline azure-pipelines-docs-integration.yml]: Keeping this so that they can be manually run for now

Comment (alexsherstinsky) [reply azure-pipelines-docs-integration.yml]: If develop and main are not needed, why not just delete them?..

File: tests/integration/test_script_runner.py [hunk 2]
```diff
@@ -23,28 +23,28 @@ class BackendDependencies(enum.Enum):
...
+    #     "util_script": "tests/integration/docusaurus/connecting_to_your_data/database/util.py",
+    #     "extra_backend_dependencies": BackendDependencies.REDSHIFT,
+    # },
+    # {
+    #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/database/redshift_yaml_example.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    #     "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples",
+    #     "util_script": "tests/integration/docusaurus/connecting_to_your_data/database/util.py",
+    #     "extra_backend_dependencies": BackendDependencies.REDSHIFT,
+    # },
```
Comment (alexsherstinsky) [inline tests/integration/test_script_runner.py:47]: If not needed, maybe best to simply delete the commented out code?..

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Left a couple of comments...

File: tests/integration/test_script_runner.py (hunk 2 above)
Comment (Shinnnyshinshin) [reply tests/integration/test_script_runner.py:47]: We will want to re-introduce these tests once the appropriate Service Account connections have been made to AWS

File: azure-pipelines-docs-integration.yml (hunk 1 above)
Comment (Shinnnyshinshin) [reply azure-pipelines-docs-integration.yml]: 👍🏼

File: tests/integration/test_script_runner.py (hunk 2 above)
Comment (Shinnnyshinshin) [reply tests/integration/test_script_runner.py:47]: These are tests that 'should' be working right now, but aren't

**Review by alexsherstinsky — APPROVED** [review]
LGTM (assuming all checks pass)...

**Pull request merged**